packages/react/src/components/Modal/Modal-story.js

```diff
@@ -153,6 +187,25 @@ storiesOf('Modal', module)
       },
     }
   )
+  .add(
+    'Title only',
```

Comment:
Any chance we could have this new example be without knobs? As we're adding stories, might make more sense to drop knob usage as they can make it harder to see how to use the component from the story.

Reply:
The knobs are still needed for heading content, etc. I think the discussion was around a story variant like this title only example, which was introduced as a story variant instead of a knob.

Reply:
Hmm, not sure I understand. Just was speaking to the `Title` only example being hard to know how to use because of the knobs. In other words, this snippet doesn't really help an external user understand how to use the component easily as it has the knob abstraction applied to it 

![image](https://user-images.githubusercontent.com/3901764/69191998-4e90ed80-0ae9-11ea-8437-cfd9d760fa0f.png)

This is something that's been coming up in our dev channels where we have been talking about not using knobs because they can make it harder to use Storybook as a reference for how to use a component.

Reply:
I see your point about the snippet readability. But I'd like to keep the ability to demonstrate different options here, esp. to keep the consistency of other modal demos.

Reply:
This isn't to suggest that we have to remove how we demonstrate these options, just was speaking to how we're choosing to demonstrate these options through knobs and how that makes things hard for existing content. Would be great if new content didn't fall into the same problem 👍

Reply:
Do you want to include that to your `v11` proposal list? I'd like to keep the knobs usage consistent across other modal stories, but we can always revisit the pattern in future.

Reply:
Definitely understand 👍 Since it's related to docs, doubt we need to wait for v11 but can be an effort over time to update.